magento/magento2
# magento/partners-magento2b2b#137: Add Company to Customers Now Online…

… grid - Added MFTF action group to CE > UI Admin grid





### Description (*)

Added MFTF Actiongroup in CE in Ui module

- AdminAddColumnToAdminGridActionGroup
- AdminOpenColumnActionDropDownForAdminGridActionGroup

### Related Pull Requests
https://github.com/magento/partners-magento2b2b/pull/146


### Fixed Issues (if relevant)

1. Fixes magento/partners-magento2b2b#137

### Manual testing scenarios (*)

1. Navigate Admin -> Customers -> Now Online
2. Check if Company column is not displayed
3. Click Columns >  Check Company checkbox
4. Check Company Column is displayed between Lastname and Email column
5. Check if Company Column is sortable
6. Click Filters
7. Check if there is Company filter
8. Type company name
9. Click Apply Filter or  press enter
10. Result should display only the filtered values

### Questions or comments


### Contribution checklist (*)
